ParabolInc/parabol
# fix(teams-limit): count only teams that have had at least 1 meeting in the last 30 days

Fixes #7915

**How to test:**

- Don't forget to add a feature flag on org
- Uncomment `const enum Threshold`
- Create 2 teams
- Set `MAX_STARTER_TIER_TEAMS = 1`
- Run 1 meeting on 1 team
- Set `STICKY_TEAM_LAST_MEETING_TIMEFRAME = 60` // 1 min
- Wait more than 1 min 
- Run 1 meeting on another team
- See that the limit was not triggered, because 1st team doesn't have any meetings within the last 1 minute
- Set `STICKY_TEAM_LAST_MEETING_TIMEFRAME = 60*15` // 15 min
- Run a meeting again, see that the limit was triggered
